python/ray/job_config.py

```diff
@@ -38,6 +49,9 @@ def __init__(
 
     def serialize(self):
         job_config = ray.gcs_utils.JobConfig()
+        if self.python_worker_setup_hook is not None:
+            job_config.python_worker_setup_hook = cloudpickle.dumps(
+                self.python_worker_setup_hook)
```

Comment:
should you raise a better error message here? often times cloudpickle will dump a bad error string. One option would be to do `cloudpickle.dumps_debug`

Reply:
(didn't see this was wip, oops)